gauge.go

```diff
@@ -103,6 +107,9 @@ func (g *Gauge) Buffer() []Point {
 				p.Bg |= AttrReverse
 			}
 
+			if g.PercentColorHighlighted != Attribute(uint16max) {
```

Comment:
> We only use up to 256 colours values, which means there are 8 digits spare space, so may be we can take advantage of it (I haven't test passing a Attribute >= 256 to termbox, not sure what will happen;))

It seems to use the default color then, but I wouldn't rely on that (because it might be terminal specific).  
So, are you suggesting that if we pass a value greater then 256, the `PercentColor` should be used? Wouldn't passing an arbitrary number greater then 256 as unexpected as using 0xFFFF?
Or how would you purpose to reset it? We could just add it to the docs and hope that the user reads the godocs if he wants to do something non-trivial.